source/leuven/wice_advanced_guide.rst

```diff
@@ -108,6 +108,26 @@ For getting a compact overview of the current state of the cluster, execute
 ``slurmtop`` on any KU Leuven Tier-2 node. Use ``slurmtop --help`` to get to
 know the functionality.
 
+.. _wice_environment_propagation:
+
+Environment propagation
+-----------------------
+
+Slurm jobs now start in a clean environment which corresponds to your login
+environment, i.e. with only those additional variables that you defined in your
+``~/.bashrc`` file. Environment variables that happen to be set in the session
+from which you submit the job are no longer propagated to the job.
+
+If needed you can modify this default behaviour with the
+`--export option <https://slurm.schedmd.com/sbatch.html#OPT_export>`__.
+When doing so, keep in mind that you will need to include the default minimal
+environment as well. To e.g. pass an additional environment variable ``FOO``
+with value ``bar``, use ``--export=HOME,USER,TERM,PATH=/bin:/sbin,FOO=bar``.
```

Comment:
Something weird is going on and I am not sure my findings are correct, but to me it seems that if you do not specify the `--export` option, Slurm defaults to `--export=HOME,USER,TERM,PATH=/bin:/sbin`. In case your job script has the `#!/usr/bin/bash` shebang, updates to `$PATH` in your `~/.bashrc` are ignored. On the other hand, if you have  `#!/usr/bin/bash -l` instead, the updates to `$PATH` are included.

Perhaps a good question is whether setting `PATH=/bin:/sbin` by default in a job is sensible. So instead of adapting the documentation, it might be better to first figure this out.

Reply:
For the time being, for sure the easiest is to inform the users that it's best to use  `#!/usr/bin/bash -l` instead of  `#!/usr/bin/bash`. Currently the use of `bash -l` is explicitly mentioned for interactive jobs (https://docs.vscentrum.be/en/latest/leuven/wice_quick_start.html#submit-to-the-interactive-partition). I will add a commit here to make clear how the shebang is supposed to look like for batch jobs.

Reply:
Done in 7baf31678d6a191c2342120ad373d6ba3aa3b23a